source/module_esolver/esolver_ks_lcao_elec.cpp

```diff
@@ -52,6 +52,15 @@ void ESolver_KS_LCAO<TK, TR>::set_matrix_grid(Record_adj& ra)
     // ModuleBase::GlobalFunc::DONE(GlobalV::ofs_running,"SEARCH ADJACENT ATOMS");
 
     // (3) Periodic condition search for each grid.
+    double dr_uniform;
```

Comment:
initialize the value

Reply:
The initialization has been added to the file